code-challenges.test.js

```diff
@@ -64,4 +101,23 @@ const accountTransactions2 = [250, -89, 100, -96]
 const accountTransactions3 = []
 // Expected output: []
 
+describe("accNum", () => {
+  it("returns an array of an accumulation of the numbers or an empty array if the input is an empty array", () => {
+    expect(accNum()).toEqual("[100, 83, 60, 51]")
+  })
+})
+
```

Comment:
You have a really good start with this! I am going to write you some pseudocode using what you already have and then using the pseudocode, try this question again.
Pseudo:
- We need a function that will iterate over an array and returns an array of the accumulating sum
- We want to declare a function called (accNum)
- We will use a for loop to iterate over the given variables (accountTransactions1, accountTransactions2, accountTransactions3)
- Our parameter will be (array)
- Next we want to declare two variables. The first being (let newArr) and another (let initialValue)
- We will then write our for loop to iterate over the length of the array, adding the given (initialValue) to (array[i]) and then using .push we are able to push the new (initialValue) into our (newArr)